docusaurus/docs/snippets/plugin-e2e-auth-project.pnpm.md

```diff
@@ -0,0 +1,25 @@
+```ts title="playwright.config.ts"
+import { dirname } from 'path';
```

Comment:
This is the only auth-project.md with `dirname` imported. I think the others will error but I don't think there's a need to differentiate between package manager for this config.ts. I think we could just put the contents of this .md file back in the getting-started.

Reply:
Yeah why not. Will fix